### PR TITLE
Add external services readme

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Add readme to external services mock

**Related issue(s)**

### CODE REVIEW

File: components/external-services-mock/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# External services mock
+
+## Overview
+
+External services mock is a test component that implement external APIs to which compass sends requests.
```
Comment (pkosiec) [inline components/external-services-mock/README.md:5]: Suggestion: `That mocks external APIs for test purposes.`

List here which mocks we have currently

File: components/external-services-mock/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Configuration
```
Comment (pkosiec) [inline components/external-services-mock/README.md:7]: Add development section how to run it locally

File: components/external-services-mock/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# External services mock
+
+## Overview
+
+External services mock is a test component that mocks external APIs for test purposes.
+
+Implemented external servies:
```
Comment (pkosiec) [inline components/external-services-mock/README.md:7]: It contains the following mocks:

File: components/external-services-mock/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* auditlog
```
Comment (pkosiec) [inline components/external-services-mock/README.md:8]: - Audit Log service

File: components/external-services-mock/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+| **APP_AUDITLOG_CLIENT_SECRET**   | The expected audit log client secret which is used in obtaining JWT token         | 
+| **APP_AUDITLOG_CLIENT_ID**       | The expected audit log client id which is used in obtaining JWT token              | 
+
+## Development
+
+To run the component locally, export all mandatory and if needed optional ENVs and run:
+
+```bash
+make build-local
+./bin/external-services-mock
```
Comment (pkosiec) [inline components/external-services-mock/README.md:30]: > "export all (...) mandatory envs"

Please use `go run` here, and provide complete command with all required ENVs (if there are any).

File: components/external-services-mock/README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# External services mock
+
+## Overview
+
+External services mock is a test component that mocks external APIs for test purposes.
+
+It contains the following mocks:
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:7]:
```suggestion
Currently, External Services Mock contains the following mocks:
```

File: components/external-services-mock/README.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+* Audit Log service
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:8]:
```suggestion
* Audit log service
```

File: components/external-services-mock/README.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+## Configuration
+
+External services mock binary allows you to override some configuration parameters. You can specify the following basic environment variables:
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:12]:
```suggestion
External Services Mock binary allows you to override some configuration parameters. You can specify the following basic environment variables:
```

File: components/external-services-mock/README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+| Name                             | Description                                                       | Default value            | 
+| ---------------------------------| ----------------------------------------------------------------- | ------------------------ | 
+| **APP_ADDRESS**                  | The address and port for the service to listen on                 | `http://127.0.0.1:8080`  | 
+
+### Audit log mock API configuration
+| Name                             | Description                                                                       | 
+| -------------------------------- | --------------------------------------------------------------------------------- | 
+| **APP_AUDITLOG_CLIENT_SECRET**   | The expected audit log client secret which is used in obtaining JWT token         |
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:21]:
```suggestion
| **APP_AUDITLOG_CLIENT_SECRET**   | The expected audit log client Secret used to obtain a JWT         | 
```

File: components/external-services-mock/README.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+| Name                             | Description                                                       | Default value            | 
+| ---------------------------------| ----------------------------------------------------------------- | ------------------------ | 
+| **APP_ADDRESS**                  | The address and port for the service to listen on                 | `http://127.0.0.1:8080`  | 
+
+### Audit log mock API configuration
+| Name                             | Description                                                                       | 
+| -------------------------------- | --------------------------------------------------------------------------------- | 
+| **APP_AUDITLOG_CLIENT_SECRET**   | The expected audit log client secret which is used in obtaining JWT token         | 
+| **APP_AUDITLOG_CLIENT_ID**       | The expected audit log client id which is used in obtaining JWT token              |
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:22]:
```suggestion
| **APP_AUDITLOG_CLIENT_ID**       | The expected audit log client ID used to obtain a JWT              | 
```

File: components/external-services-mock/README.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+### Audit log mock API configuration
+| Name                             | Description                                                                       | 
+| -------------------------------- | --------------------------------------------------------------------------------- | 
+| **APP_AUDITLOG_CLIENT_SECRET**   | The expected audit log client secret which is used in obtaining JWT token         | 
+| **APP_AUDITLOG_CLIENT_ID**       | The expected audit log client id which is used in obtaining JWT token              | 
+
+## Development
+
+To run the component locally run:
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:26]:
```suggestion
Use this command to run the component locally:
```

File: components/external-services-mock/README.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+```bash
+export APP_AUDITLOG_CLIENT_SECRET=client-secret
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:29]:
```suggestion
export APP_AUDITLOG_CLIENT_SECRET={CLIENT_SECRET}
```

File: components/external-services-mock/README.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+export APP_AUDITLOG_CLIENT_ID=client-id
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:30]:
```suggestion
export APP_AUDITLOG_CLIENT_ID={CLIENT_ID}
```

File: components/external-services-mock/README.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
+# External Services Mock
+
+## Overview
+
+External Services Mock is a component that mocks external APIs for test purposes. To learn more about tests that use the External Services Mock, read [this](https://github.com/kfurgol/compass/blob/master/tests/director/external-services-mock-integration/README.md) document.
+
+It contains the following mocks:
```
Comment (klaudiagrz) [inline components/external-services-mock/README.md:7]:
```suggestion
External Services Mock contains the following mocks:
```

**Pull request merged**